src/dashboard/cd-dashboard-children.vue

```diff
@@ -24,9 +64,16 @@
     margin-left: auto;
     min-height: 100%;
     display:flex;
+    flex-direction: column;
```

Comment:
Sorry for third random review, just noticed this in a different PR. 😅  Seems like there's no need for this being flex now that it's column.

Would also be curious if `min-height` is needed here too. I think it is for the moment, and I think I know why, but we should be able to fix that to simplify things. Don't worry about it for now. I'll probably do an overall audit of the CSS anyway and see where we can simplify things. We'll see. 😛

Reply:
oh the flex column is for the elements inside the children to be displayed stacked 
-
-
-
^ like that

Reply:
Column stacking is the default behaviour for block elements, which is all that's used inside. It's no biggie, but it should work without flex unless I'm missing something.

Reply:
yeah all good. just fixed that there